src/views/Zap/__tests__/ZapStakeAction.unit.test.tsx

```diff
@@ -321,6 +176,5 @@ describe("<ZapStakeAction/> Not on Mainnet", () => {
     expect(
       screen.getByText("Zaps are only available on Ethereum Mainnet. Please switch networks."),
     ).toBeInTheDocument();
-    expect(screen.getByText("Enter Amount").closest("button")).toBeDisabled();
```

Comment:
We *should* be testing that the button is disabled

Reply:
the behavior redirects you away from the view though right? So there would be no element on the screen to test.

Reply:
To test, go to the Zap screen, then switch networks. You'll see a message appear, but youll be redirected away from zap

Reply:
True. Consider this a failsafe. That behaviour is at a higher level (app or router?), so this component does not actually redirect. I think we should still test it, in case someone is able to get through.

Reply:
Sounds good. I'm just not sure how we would test it in this context though. Since we're expecting a value on the screen, and the screen is now different. 
If you could figure out a way to get this test passing, i'm all for keeping it. But technically using this method, the button isn't on the screen so this step in the test will fail.

Reply:
have a look at the render method in testUtils.tsx. Even though we are only rendering the ZapStakeAction view, our render method wraps providers around it, including BrowserRouter. 
```    <Web3ContextProvider>
      <ReactQueryProvider>
        <Provider store={store}>
          <I18nProvider i18n={i18n}>
            <BrowserRouter basename={"/"}>
              <ThemeProvider theme={lightTheme}>
                <CssBaseline />
                {children}
              </ThemeProvider>
            </BrowserRouter>
          </I18nProvider>
        </Provider>
      </ReactQueryProvider>
    </Web3ContextProvider>
    ```

Reply:
IIRC the test was passing with the line that you removed. Try it?

Reply:
You're right. I swear it stopped working for a min, which is why i turned it off, but it does appear to work. I've added it back. If we get intermittent test failures because of flakiness (i.e. screen redirects before this line evaluates) , we may want to disable, but its working for now. Thx for reviewing!